libraries/standard/http/include/http_client.h

```diff
@@ -423,19 +457,30 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * parameter pRequestBodyBuf over the transport. The response is received in
  * #HTTPResponse_t.
  *
+ * The application should close the connection with the server if any
+ * HTTP_SECURITY_ALERT_X errors are returned. Any error found in parsing is
+ * considered a malformed response and therefore a security alert.
```

Comment:
The "Any error found in parsing is considered a malformed response and therefore a security alert" seems incomplete and or misleading about the quality of the library. 
What about errors arising from internal failures during parsing? Would that be also elevated as a "security alert"? 
These questions are the type of misleading interpretations of the sentence

Reply:
Good point deleting.